static/title.less

```diff
@@ -0,0 +1,44 @@
+ul.title-social{
+	float: right;
+	margin-top: -47px;
+	line-height: 1.6;
+	margin-bottom: @gutter;
+    list-style-type: disc;
+    display: none;
+
+    @media screen and (min-width: @breakpoint) {
+    	display: unset;
+    }
+
+	> li{
```

Comment:
Your spacing/indentation throughout here is really aggressive. :) You'll find that other devs will give you a hard time about stuff like this, so you might fix it all so it's more readable.